File: ProyectoMiLunaCielo/frontEnt/src/app/Model/inventario.ts
export interface Response {
  statusCode:       number;
  idConsulta:       number;
  codigoRespuesta:  number;
  mensajeRespuesta: string;
  fechaConsulta:    Date;
  datos:            InventarioDto[];
  claims:           null;
  base64:           null;
}

export interface InventarioDto {
  idtipoInventario:  string;
  codigo:            string;
  cantidad:          number;
  oficina:           string;
  descripcion:       string;
  observaciones:     string;
  area:              string;
  fechaCreacion:     Date;
  fechaModificacion: Date;
  estado:            string;
}

File: ProyectoMiLunaCielo/BackEnd2023/BackEnd2023/Entidades/bd.Inventarios/inventario.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackEnd2023.Entidades.bd.Inventarios
{
    public class inventario
    {
        [Key]
        public int Id { get; set; }

        [Column("id_tipoInventario")]
        public string IdtipoInventario { get; set; }
        public string codigo { get; set; }
        public long cantidad { get; set; }
        public string oficina { get; set; }
        public string descripcion { get; set; }

        public string observaciones { get; set; }
        public string area { get; set; }
        public DateTime fechaCreacion { get; set; }

        public DateTime fechaModificacion { get; set; }
        public string estado { get; set; }
    }
}
